pkg/clients/account/client.go

```diff
@@ -4,4 +4,7 @@ import "context"
 
 type Client interface {
 	GetAccount(ctx context.Context) (string, error)
+
```

Comment:
Maybe a comment above `GetAccount` specifying that it's for getting the account ID?

Reply:
Yeah would be a nice drive-by change.